pkg/apply/applier.go

```diff
@@ -180,14 +197,17 @@ func (a *Applier) Run(ctx context.Context, invInfo inventory.InventoryInfo, obje
 			InventoryPolicy:        options.InventoryPolicy,
 		}
 		// Build list of apply validation filters.
-		applyFilters := []filter.ValidationFilter{}
-		if options.InventoryPolicy != inventory.AdoptAll {
-			applyFilters = append(applyFilters, filter.InventoryPolicyApplyFilter{
+		applyFilters := []filter.ValidationFilter{
```

Comment:
This change is a net win, IMO. It simply moves the if statement into the InventoryPolicyApplyFilter.

Reply:
extracted to https://github.com/kubernetes-sigs/cli-utils/pull/541